README.md

```diff
@@ -42,16 +41,15 @@ Ruby setup, you might need to run `bundle exec citizen` instead of just `citizen
 
 ### Deploying
 
-* [CircleCI](https://circleci.com/gh/codeforamerica/michigan-benefits) is currently set up to 
+* [CircleCI](https://circleci.com/gh/codeforamerica/michigan-benefits) is currently set up to
```

Comment:
Minor: Do we have a badge icon for the `README` file to display build status? I figure since you're editing this file already, might as well add it in before merging this? What do you think?